tst/EnergyPlus/unit/UnitaryHybridAirConditioner.unit.cc

```diff
@@ -0,0 +1,433 @@
+#include <EnergyPlus/HeatBalanceManager.hh>
+#include <ConfiguredFunctions.hh>
+#include <EnergyPlus/MixedAir.hh>
+#include <EnergyPlus/DataContaminantBalance.hh>
+#include <EnergyPlus/DataAirLoop.hh>
+#include <EnergyPlus/DataAirSystems.hh>
+#include <EnergyPlus/DataGlobals.hh>
+#include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/HeatBalanceManager.hh>
+#include <EnergyPlus/Humidifiers.hh>
+#include <EnergyPlus/ScheduleManager.hh>
+#include <EnergyPlus/SizingManager.hh>
+#include <EnergyPlus/DataEnvironment.hh>
+#include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/DataLoopNode.hh>
+#include <EnergyPlus/DataZoneEnergyDemands.hh>
+#include <EnergyPlus/DataZoneControls.hh>
+#include <EnergyPlus/Psychrometrics.hh>
+#include <fstream>
+using namespace EnergyPlus::MixedAir;
+using namespace EnergyPlus::DataContaminantBalance;
+using namespace EnergyPlus::DataAirLoop;
+using namespace EnergyPlus::DataAirSystems;
+using namespace EnergyPlus::DataSizing;
+using namespace EnergyPlus::DataHeatBalance;
+using namespace EnergyPlus::ScheduleManager;
+using namespace EnergyPlus::DataEnvironment;
+using namespace EnergyPlus::DataZoneEquipment;
+using namespace EnergyPlus::DataLoopNode;
+using namespace EnergyPlus::DataZoneEnergyDemands;
+using namespace EnergyPlus::DataZoneControls;
+using namespace EnergyPlus::HeatBalanceManager;
+using namespace EnergyPlus::Humidifiers;
+using namespace EnergyPlus::SizingManager;
+
+using namespace EnergyPlus::DataContaminantBalance;
+using namespace EnergyPlus::MixedAir;
+using namespace EnergyPlus;
+using namespace EnergyPlus::SizingManager;
+using EnergyPlus::CurveManager::CurveValue;
+using EnergyPlus::CurveManager::GetNormalPoint;
+using EnergyPlus::CurveManager::GetCurveName;
+using EnergyPlus::Psychrometrics::PsyHFnTdbRhPb;
+using EnergyPlus::Psychrometrics::PsyRhFnTdbWPb;
+using EnergyPlus::Psychrometrics::PsyWFnTdbRhPb;
+using namespace EnergyPlus::ScheduleManager;
+using EnergyPlus::HybridEvapCoolingModel::CMode;
+using EnergyPlus::HybridEvapCoolingModel::CSetting;
+using EnergyPlus::HybridEvapCoolingModel::Model;
+using namespace EnergyPlus::HybridUnitaryAirConditioners;
+
+namespace EnergyPlus {
+	std::vector<std::string> getAllLinesInFile2(std::string filePath) {
+		std::ifstream infile(filePath);
+		std::vector<std::string> lines;
+		std::string line;
+		while (std::getline(infile, line))
+		{
+			lines.push_back(line);
+		}
+		return lines;
+	}
+
+	std::vector<std::string> parseLine(std::string line)
+	{
+		std::vector<std::string> vect;
+		std::stringstream ss(line);
+		std::string token;
+
+		while (std::getline(ss, token, ',')) {
+			vect.push_back(token);
+		}
+		return vect;
+	}
+
+
+	TEST_F(EnergyPlusFixture, Test_UnitaryHybridAirConditioner_Unittest) {
+		using namespace InputProcessor;
+		std::vector<std::string> snippet = getAllLinesInFile2(configured_source_directory() + "/datasets/UnitaryHybridUnitTest_MuntersEPX5000.idf");

```

Comment:
This line needs to be changed now, as the test file here is in the unit test directory, not in `datasets/`.

Reply:
this is addressed, at least I think it is, i had some trouble pushing it up.